tools/fpgabist/fpgabist

```diff
@@ -84,4 +84,15 @@ if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     bist_common.global_arguments(parser)
     args = parser.parse_args()
+    reqs = []
+    for cmd in bist_common.REQ_CMDS:
+        if(not bist_common.is_command_in_path(cmd)):
+            reqs.append(cmd)
+    if reqs:
+        req = ', '.join([r for r in reqs])
+        print "Unable to find {} in system PATH.".format(req)
+        print "Please see README for required commands"
+        print "Terminating BIST."
+        sys.exit(-1)
```

Comment:
FYI. It doesn't need to be changed.
* Default return code for "No such file or directory" is `2`.
* You can pass any text to `exit()` and print it on exit, signaling `1` as exit code.
```
Help on built-in function exit in module sys:

exit(...)
    exit([status])

    Exit the interpreter by raising SystemExit(status).
    If the status is omitted or None, it defaults to zero (i.e., success).
    If the status is an integer, it will be used as the system exit status.
    If it is another kind of object, it will be printed and the system
    exit status will be one (i.e., failure).
```